examples/integration_wgpu/src/main.rs

```diff
@@ -91,11 +93,16 @@ pub fn main() {
         #[cfg(not(target_arch = "wasm32"))]
         let needed_limits = wgpu::Limits::default();
 
+        let capabilities = surface.get_capabilities(&adapter);
+
         (
-            surface
-                .get_supported_formats(&adapter)
-                .first()
+            capabilities
+                .formats
+                .iter()
+                .filter(|format| format.describe().srgb)
```

Comment:
This is cool! Hopefully it fixes inaccurate colors for some!

Reply:
Haha yeah the API is improving a lot, I was surprised too! Thank you.